SUMMARY.md

```diff
@@ -144,17 +144,15 @@
  * [Transfer Complete Event](bxml/callBacks/transfer.md)
 
 ## BXML Guides
-* [BXML Guides](bxml/guides.md)
- * [BXML Concepts and Vocabulary](bxml/bxmlConcepts.md)
- * [BXML Overview](bxml/bxmlOverview.md)
```

Comment:
This is here twice, here & above

Reply:
These are deletions. I reworked the sidebar regarding bxml. This is how it'll look

```
BXML FOR VOICE
BXML Overview
BXML Concepts and Vocabulary
BXML Verbs
BXML Callbacks
BXML GUIDES
BXML Building Blocks
```